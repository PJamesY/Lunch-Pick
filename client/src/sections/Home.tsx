import React from 'react';

import { StyledSection } from '../styledComponent/Section';

const Home = function Home() {
  return <StyledSection id="home">Home</StyledSection>;
};

export default Home;
