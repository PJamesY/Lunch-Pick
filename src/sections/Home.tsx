import React from 'react';
import styled from 'styled-components';

import { StyledSection } from '../styledComponent/Section';
import hello from '../asset/hello.gif';

const StyledImg = styled.img`
  width: 100%;
`;

const Home = function Home() {
  return (
    <StyledSection id="home">
      <StyledImg src={hello} alt="background" />
    </StyledSection>
  );
};

export default Home;
