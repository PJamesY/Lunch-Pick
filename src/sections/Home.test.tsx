import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('Home section', () => {
  it('should match snapShot', () => {
    expect(shallow(<Home />)).toMatchSnapshot();
  });
});
