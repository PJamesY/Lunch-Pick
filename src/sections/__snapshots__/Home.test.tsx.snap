// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home section should match snapShot 1`] = `ShallowWrapper {}`;
